tools/building.py

```diff
@@ -528,6 +528,8 @@ def lld_flags_for_executable(external_symbol_list):
     if external_symbol_list:
       # Filter out symbols external/JS symbols
       c_exports = [e for e in c_exports if e not in external_symbol_list]
+    if Settings.STANDALONE_WASM and Settings.EXPECT_MAIN:
+      c_exports.remove('main')
```

Comment:
this seems surprising - if we expect main, why remove it?

Reply:
Because in standalone more if you have a `main` the way to run it is via `_start` which is the default value for `--entry`.   standalone builds never need to export main directly.

Reply:
Perhaps `EXPECT_MAIN` could be renamed to `EXPECT_ENTRY_POINT` to make this less confusing but I'm not sure it worth it.

Reply:
Thanks, I see. Yeah, maybe not worth it right now, but it seems worth it eventually. For now at least please add a comment here as this is quite confusing I think. And maybe a TODO on `EXPECT_MAIN` in settings.js?